### PR TITLE
Entries for my Frontiers paper

### PR DESCRIPTION
**Pull request by jonaskuResearch**
For the authors Vargas_PA, Queralta_JP, Karaguzel_TA, Vieira_SM, Ramos_RP, and Dosieah_GY, I did not find any authoritative sources on what is their first and last name. I have used my best judgement according to the information provided by publishers, Google Scholar, and DBLP.

### CODE REVIEW

File: author.bib [hunk 1]
```diff
@@ -326,11 +341,13 @@ @string{ Gaudiano_P
 @string{ Gautrais_J           = "Gautrais, Jacques" }
 @string{ Gelatt_CDJr          = "Gelatt, Jr., C. Daniel" }
 @string{ Geman_S              = "Geman, Stuart" }
+@string{ Gent_IP              = "Gent, Ian P." }
```
Comment (glegarda) [inline author.bib:344]: Should appear after Gendreau_M (Ganguli_M is misplaced; should appear after Gambin_T)

File: author.bib [hunk 2]
```diff
@@ -398,6 +417,7 @@ @string{ Heaton_J
 @string{ Hebert_M             = "Hebert, Martial" }
 @string{ Hecker_JP            = "Hecker, Joshua P." }
 @string{ HeidrichMeisner_V    = "Heidrich-Meisner, Verena" }
+@string{ Heinerman_J          = "Heinerman, Jacqueline" }
```
Comment (glegarda) [inline author.bib:420]: Should appear after Heil_R, not before

File: author.bib [hunk 3]
```diff
@@ -573,6 +596,7 @@ @string{ Malavolta_I
 @string{ Marcuse_S            = "Marcuse, Sophie" }
 @string{ Marjan_M             = "Marjan, Mernik" }
 @string{ Markovitch_S         = "Markovitch, Shaul" }
+@string{ Mariano_P            = "Mariano, Pedro" }
```
Comment (glegarda) [inline author.bib:599]: Should appear after Marcuse_S

File: author.bib [hunk 4]
```diff
@@ -886,13 +918,15 @@ @string{ Tan_X
...
 @string{ Taylor_R             = "Taylor, Russell" }
+@string{ Taylor_T             = "Taylor, Tim" }
 @string{ Teller_AH            = "Teller, Augusta H." }
 @string{ Teller_E             = "Teller, Edward" }
 @string{ Thayer_D             = "Thayer, David" }
 @string{ Theraulaz_G          = "Theraulaz, Guy" }
 @string{ Thiele_L             = "Thiele, Lothar" }
 @string{ Thienel_J            = "Thienel, Julie" }
 @string{ Tibshirani_R         = "Tibshirani, Robert"}
+@string{ Tierney_K            = "Tierney, Kevin" }
```
Comment (glegarda) [inline author.bib:929]: Both this one and Tibshirani_R should appear after Thrun_S

File: journal.bib [hunk 5]
```diff
@@ -93,6 +97,7 @@ @string{ RAIRO
 @string{ RMP         = "Reviews of Modern Physics" }
 @string{ ROB         = "Robotica" }
 @string{ RAS         = "Robotics and Autonomous Systems" }
+@string{ RSOS        = "Royal Society Open Science" }
```
Comment (glegarda) [inline journal.bib:100]: Missing in journal_short.bib

File: bibliography.bib [hunk 6]
```diff
@@ -7136,9 +7136,508 @@ @Article {ProHsiKum2017IEEETR
...
+    year         = {2014},
+    pages        = {153--178},
+    publisher    = pub_MITP,
+    address      = Boston_MA_USA,
+    doi          = {10.7551/mitpress/8493.003.0008},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {GhaKucGarBir2023icra,
```
Comment (glegarda) [inline bibliography.bib:7385]: Maybe add a comment here (besides the note) so we don't forget to update the entry once a decision has been made?

**Review by glegarda — CHANGES_REQUESTED** [review]

File: journal.bib (hunk 5 above)
Comment (jonaskuResearch) [reply journal.bib:100]: Added to journal_short.bib

File: bibliography.bib (hunk 6 above)
Comment (jonaskuResearch) [reply bibliography.bib:7386]: Added a comment

**Pull request merged**